tests/e2e/e2e.go

```diff
@@ -136,10 +136,10 @@ func setupSuite() {
 
 	ginkgo.By("Initializing all nodes")
 	nodeList, err := c.CoreV1().Nodes().List(metav1.ListOptions{})
-	framework.ExpectNoError(err)
+	framework.ExpectNoError(err, "list nodes failed")
```

Comment:
(unimportant) We can unify the format of `failed to xxx` with `xxx failed`.

Reply:
actually this PR uses `failed to xxx` all the time, and it's not so easy to change because there are so many lines...